README.md

```diff
@@ -29,6 +29,55 @@ Run:
 ./index.js couch-import --source-couch-db=[couch-db-url]/_changes --target-registry=[target-registry-url] --shared-fetch-secret=[password-from-console]
 ```
 
+## Importing From npm Orgs
+
+### 1. Create matching organization in npm Enterprise
+
+You should create an organization in npm Enterprise that matches the name of
+the organization that you wish to migrate from in npm Orgs.
+
+As an example, if you want to migrate from your organization `@babel` on the
+public registry, create an organization named `babel` in npm Enterprise.
+
+### 2. Create an authorization token
+
+To migrate from npm Orgs, you need an authorization token from the public
+registry.
+
+1. visit `https://www.npmjs.com`.
+2. in the account drop down in the upper right, choose the option "Tokens".
+3. click "Create New Token", and copy this somewhere safe for later.
+
+### 3. Login to your npm Enterprise Instance
+
+```
+npm config set registry https://registry.my-instance.npme.io
+npm login
+```
+
+### 4. Create a text file containing the packages you wish to migrate
+
+_Note: you can only publish scoped modules to your private registry._
+
+The text file should look something like this:
```

Comment:
Can this have commas or other line separators, or does each entry need a new line?

Reply:
Each entry should be a new line.